backend/routes/group.py

```
@@ -1,11 +1,16 @@
 from fastapi import APIRouter, Depends, HTTPException
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)